PSOperationsTests/PSOperationsTests.swift

```diff
@@ -973,4 +1006,40 @@ class PSOperationsTests: XCTestCase {
         XCTAssertEqual(1, opQ.operationCount)
         XCTAssertTrue(op.waitCalled)
     }
+    
+    func testOperationsNotExecuting() {
+        
+        var opCount = 0
```

Comment:
This is probably better named `passCount`. Is there a way to also accumulate a `failCount` or a `totalCount` and make sure that `failCount < 25` or something so that you don't have 50,000 failures to get 25,000 passes.